### ISSUE TITLE
Meteor 1.3: Uncaught Error: use "new" to construct a Mongo.Collection

### ISSUE THREAD

**Issue by LucNaterop** [GroundMeteor/db#158]
When I create a collection
`Parties = new Ground.Collection('parties', { connection: null });`

**Comment by zevsuld** [comment]
I have same issue. @LucNaterop did you fixed it?

**Comment by Andrew1431** [comment]
So I installed this package and STARTED getting this error. Didn't get it before

`meteortoys:allthings`

Only problem is, that means it could be one of any of the following packages.

```accounts-base                       added, version 1.2.4
dburles:mongo-collection-instances  added, version 0.3.4
ddp-rate-limiter                    added, version 1.0.2
email                               added, version 1.0.10
gwendall:body-events                added, version 0.1.6
lai:collection-extensions           added, version 0.1.4
localstorage                        added, version 1.0.7
meteortoys:allthings                added, version 2.3.1
meteortoys:authenticate             added, version 2.1.0
meteortoys:autopub                  added, version 2.1.0
meteortoys:blueprint                added, version 2.1.0
meteortoys:email                    added, version 2.1.0
meteortoys:hotreload                added, version 2.1.0
meteortoys:listen                   added, version 2.1.0
meteortoys:method                   added, version 3.0.2
meteortoys:pub                      added, version 3.0.2
meteortoys:result                   added, version 2.1.0
meteortoys:shell                    added, version 2.1.0
meteortoys:status                   added, version 2.1.0
meteortoys:sub                      added, version 2.1.0
meteortoys:throttle                 added, version 2.1.0
meteortoys:toykit                   added, version 2.2.1
msavin:jetsetter                    added, version 1.5.2
msavin:mongol                       added, version 1.6.2
rate-limit                          added, version 1.0.2
service-configuration               added, version 1.0.7
session                             added, version 1.1.3```

**Comment by Jahdrien** [comment]
Try this:
Remove both ground:db and meteortoys:allthings then add meteortoys:allthings then ground:db in this order.
I have this error with any package touching collection installed after ground:db...

**Comment by Andrew1431** [comment]
Thank you! I'll try that out.

**Comment by raix** [comment]
https://github.com/GroundMeteor/db/issues/153#issuecomment-206125703

**Comment by LucNaterop** [comment]
@raix Thanks for the awesome work. The issue is resolved `2.0.0-alpha.2 `
